tests/libnet/cluster.go

```diff
@@ -3,6 +3,9 @@ package libnet
 import (
 	"fmt"
 
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
This dependency is weird here... This file couldn't care less about ginkgo / gomega.